src/models/campaign.ts

```diff
@@ -28,6 +28,7 @@ export interface Data {
     antigens: Antigen[];
     antigensDisaggregation: AntigensDisaggregation;
     targetPopulation: Maybe<TargetPopulation>;
+    attributeValues: AttributeValue[];
```

Comment:
I think it's better to abstract this behind the campaign class, and callers can ask for specific information. Here, exposing `campaign.dashboardId` seems convenient: 1214dbaf